Extensions/AadIssuerValidator.cs

```diff
@@ -23,29 +21,29 @@ public static class AadIssuerValidator
         /// <returns>The <c>issuer</c> if it's valid, or otherwise <c>SecurityTokenInvalidIssuerException</c> is thrown</returns>
         public static string ValidateAadIssuer(string issuer, SecurityToken securityToken, TokenValidationParameters validationParameters)
         {
-            JwtSecurityToken jwtToken = securityToken as JwtSecurityToken;
+            var jwtToken = securityToken as JwtSecurityToken;
```

Comment:
not sure what's the point of replacing the types by var?

Reply:
Frankly, it is a coding style I prefer. From code reading perspective you already see, that is JwtSecurityToken due to cast. Compiler will determine the type on the end. It is not required, it is more of a preference (more [inline with .NET framework](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/var) coding styles)

Reply:
@bgavrilMS to decide on this ones :)

Reply:
I think our style is to prefer using 'var' when the type on right is apparent, which is the case here. The style of the samples does not have to match the style of the library, but it seems like a reasonable approach.

Reply:
Also, VS 2019 now proposes 3 rules to deal with var

1. Don't use var for standard types (e.g. int i = 3) - not sure if this extends to string, but it makes sense.
2. Use or Don't use var when type of the right is apparent (e.g. var x = new C() is ok, var y = obj.GetStuff() is not ok)
3. Use or Don't use var everywhere else.